### DISCUSSION TITLE
guide/sitemap/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#278] [docs-comment]
# guide/sitemap/

https://theme-plume.vuejs.press/guide/sitemap/

**Comment by SingleMoonlight** [comment]
配置 hostname 后，不管是本地还是 GitHub Action，build 貌似都没有生成 sitemap.xml 文件，请问是否需要其他配置？

**Reply by pengzhanbo** [reply]
`vuepress@2.0.0-rc.17` 这个版本有点问题，`app.env.isBuild` 在构建阶段的值不正确，导致了 sitemap 没有生成。 `vuepress@2.0.0-rc.18` 已修复了这个问题，稍后我会将主题的 vuepress 最低依赖版本 升级。
